packages/syft/src/syft/core/node/common/util.py

```diff
@@ -115,7 +115,7 @@ def upload_result_to_s3(
         asset_name=asset_name,
         dataset_name=dataset_name,
         node_id=domain_id,
-        dtype=data_dtype,
```

Comment:
Q: Could we delete the `data_type`? (it is used anywhere else?)

Reply:
Good catch! Yup, we must delete it.

Reply:
Solved by: [54a6c1e](https://github.com/OpenMined/PySyft/pull/6997/commits/54a6c1e92d38de1bc27952bf332c902ec0a53a67)